### DISCUSSION TITLE
对接夸克云盘后提示ha3 service: [HA3_SERVER_ERR]

### DISCUSSION THREAD

**Discussion by hainc** [AlistGo/alist#591] [Q&A] [answered]
如下图，如何解决？
![QQ截图20220222145438](https://user-images.githubusercontent.com/56694990/155078656-540ae29a-493e-4e7a-b9fb-ef3b5feecb54.png)
![QQ截图20220222145444](https://user-images.githubusercontent.com/56694990/155078660-d3ee2462-7b2b-4897-9eb0-43a0368a0918.png)

**Comment by xhofe** [comment, marked as answer]
cookie错误 正确的cookie应该是`__kps=`开头。

**Reply by hainc** [reply]
> cookie错误 正确的cookie应该是`__kps=`开头。

可是我edge google qq 火狐几个浏览器按照官方文档教程所示的方式获取的都不是`__kps=`开头，而且使用该cookie保存账号时也提示成功了，前台就一直提示上面所示的那个错误
![QQ截图20220227194502](https://user-images.githubusercontent.com/56694990/155881083-e88f033d-3e16-46e0-8b40-360d6b05f05a.png)

**Reply by SakurajimMai** [reply]
我也出现这样的错误，请问怎么解决的啊

**Reply by hainc** [reply]
> 我也出现这样的错误，请问怎么解决的啊

我的未解决

**Comment by SakurajimMai** [comment]
我也出现这样的错误，请问怎么解决的啊

**Comment by 631894570** [comment]
cookie      `_UP_A4A_11_=xxx; _UP_D_=pc; _UP_30C_6A_=xxx; _UP_TS_=xxx; _UP_E37_B7_=xxx; _UP_TG_=xxx; _UP_335_2B_=1; __kps=xxx; __puus=xxx; __pus=xxx; __kp=xxx `  
将`__kps=xxx; __puus=xxx; __pus=xxx; __kp=xxx1 `  放到最前面并在后面添加   `;`  变成`__kps=xxx; __puus=xxx; __pus=xxx; __kp=xxx ; _UP_A4A_11_=xxx; _UP_D_=pc; _UP_30C_6A_=xxx; _UP_TS_=xxx; _UP_E37_B7_=xxx; _UP_TG_=xxx; _UP_335_2B_=1  `
提交可用  但是后台不显示work  可直链下载
![test](https://pan.yccz.top:2443/d/%E5%A4%B8%E5%85%8B%E7%BD%91%E7%9B%98/1.png)

**Reply by hainc** [reply]
按照您的示例已成功，谢谢
